deps/Makefile

```diff
@@ -1752,6 +1752,7 @@ $(build_shlibdir)/libsuitesparse_wrapper.$(SHLIB_EXT):  $(SUITESPARSE_OBJ_TARGET
 endif
 
 $(build_shlibdir)/libsuitesparse_wrapper.$(SHLIB_EXT): $(SRCDIR)/SuiteSparse_wrapper.c
```

Comment:
Not sure which one is better, this is simply following the convention used by [`staged-install`](https://github.com/JuliaLang/julia/blob/b61f6603d059b0a49df5c5551d5a1a663a7ae504/deps/Makefile#L235) (and `make-install`) which are used by most other deps.